stock_picking_batch_extended/__manifest__.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2012-2014 Alexandre Fayolle, Camptocamp SA
+# Copyright 2018-2020 Tecnativa - Carlos Dauden
+# Copyright 2023 FactorLibre - Boris Alias
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock batch picking extended",
+    "summary": "Allows manage a lot of pickings in batch",
+    "version": "16.0.1.0.0",
+    "author": "Camptocamp, "
+    "Tecnativa, "
+    "FactorLibre, "
+    "Odoo Community Association (OCA)",
+    "development_status": "Mature",
+    "maintainers": ["gurneyalex", "carlosdauden", "i-vyshnevska", "BorisFL"],
```

Comment:
Don't put yourself as maintainer if you don't plan to maintain the module.

Reply:
I just fixed it.